### PR TITLE
Updated some constructor classes and added from/sender settings.

### PR DESCRIPTION
**Pull request by xtech86**

### CODE REVIEW

File: component/site/libraries/commonfunctions.php [hunk 1]
```diff
@@ -328,16 +331,32 @@ public static function notifyAuthorPublished($event){
 
 		// mail function
 		$mail = JFactory::getMailer();
-		$mail->setSender(array( 0 => $adminEmail, 1 => $adminName ));
-		$mail->addRecipient($authoremail);
+		$sender_config = $params->get('sender_config');
```
Comment (GeraintEdwards) [inline component/site/libraries/commonfunctions.php]: We need a default value 
`$sender_config = $params->get('sender_config',0);`

File: component/admin/language/en-GB.com_jevents.ini [hunk 2]
```diff
@@ -1366,4 +1366,16 @@ JEV_SMARTPHONE_ALWAYS_AS_LIST_DESC="When enabled will always show week and month
 
 JEV_CANNOT_DISPLAY_SAVED_EVENT_ON_THIS_MENU_ITEM="It is not possible to display the event you have saved on this menu item probably due to menu item contstraints - you should consider relaxing the menu item constraints or using the 'Missing Events Finder' plugin"
 JEV_LOAD_JOOMLA_BOOTSTRAP_CSS_IN_CALENDAR_MODULE="Load Joomla Boostrap CSS file in calendar module"
-JEV_LOAD_JOOMLA_BOOTSTRAP_CSS_IN_CALENDAR_MODULE_DESC="Should the mini-calendar module load the core Joomla bootstrap.css file - disable this option if your template loads its own copy."
\ No newline at end of file
+JEV_LOAD_JOOMLA_BOOTSTRAP_CSS_IN_CALENDAR_MODULE_DESC="Should the mini-calendar module load the core Joomla bootstrap.css file - disable this option if your template loads its own copy."
+
+JEV_SENDER_CONFIG="Send emails from:"
+JEV_SENDER_CONFIG_DESC="This is what determins the sender name and email address of event notifications being sent by JEvents Core"
```
Comment (GeraintEdwards) [inline component/admin/language/en-GB.com_jevents.ini]: determines

File: component/site/libraries/commonfunctions.php [hunk 3]
```diff
@@ -377,7 +396,23 @@ public static function sendAdminMail( $adminName, $adminEmail, $subject='', $tit
 		
 		// mail function
 		$mail = JFactory::getMailer();
-		$mail->setSender(array( 0 => $adminEmail, 1 => $adminName ));
+		$sender_config = $params->get('sender_config');
```
Comment (GeraintEdwards) [inline component/site/libraries/commonfunctions.php]: as above

File: component/site/libraries/commonfunctions.php [hunk 4]
```diff
@@ -377,7 +396,23 @@ public static function sendAdminMail( $adminName, $adminEmail, $subject='', $tit
...
+
+		} elseif ($sender_config == 1) {
+
+			$mail->setSender(array(0 => $config->mailfrom, 1 => $config->fromname));
+
+		} else {
+			$mail->setSender(array(0 => $params->get('sender_email'), 1 => $params->get('sender_name')));
+		}
+
+		if ($params->get('email_replyto') == 0) {
```
Comment (GeraintEdwards) [inline component/site/libraries/commonfunctions.php]: default value again please

File: component/site/libraries/filters.php [hunk 5]
```diff
@@ -199,6 +151,55 @@ function __construct($item, $filterpath=false){
 
 	}
 
+
```
Comment (GeraintEdwards) [inline component/site/libraries/filters.php]: Is this just a copy - Git Diff is not showing me a line by line comparison?

File: component/site/libraries/helper.php [hunk 6]
```diff
@@ -2584,7 +2584,7 @@ function getFilterValues()
 				{
 					$filtervars = get_object_vars($filtervars);
 				}
-				var_dump($filtervars);
+				//var_dump($filtervars);
```
Comment (GeraintEdwards) [inline component/site/libraries/helper.php]: thanks

File: component/site/libraries/filters.php (hunk 5 above)
Comment (xtech86) [inline component/site/libraries/filters.php]: yup, to make it more readable that's all I place the constructor at the top.

File: component/site/libraries/commonfunctions.php (hunk 4 above)
Comment (xtech86) [inline component/site/libraries/commonfunctions.php]: Hangs head in shame, change coming through shortly.

File: component/admin/language/en-GB.com_jevents.sys.ini [hunk 7]
```diff
@@ -83,3 +83,11 @@ JEVENTS="JEvents"
 
 COM_JEVENTS_LIST_DEFAULT_TITLE="List of Events"
 COM_JEVENTS_LIST_DEFAULT_DESC="List of Events where you can choose the columns to display"
+
+; Float Theme language strings ##
+PLG_INST_JEVENTS_JEVFLOAT="JEvents Float Layout Installed"
+PLG_INST_JEVENTS_JEVFLOAT_SUCC2="You have successfully installed the JEvents Float Layout"
+PLG_INST_JEVENTS_JEVFLOAT_SUCC1="You have successfully update the JEvents Float Layout"
+JEV_EVENT_STARTED="Event Started"
```
Comment (GeraintEdwards) [inline component/admin/language/en-GB.com_jevents.sys.ini]: Why did you move this from .ini to .sys.ini ?

Comment (xtech86) [inline component/admin/language/en-GB.com_jevents.sys.ini]: Because sys.ini is called on Install whereas .ini is not.
